evap/student/tests/test_views.py

```diff
@@ -60,6 +60,20 @@ def setUpTestData(cls):
 
         cls.evaluation.general_contribution.questionnaires.set([cls.top_general_questionnaire, cls.bottom_general_questionnaire])
 
+    @generate_fixtures
+    def generate_fixtures(self):
+        with_confirmation = self.app.get(self.url, user=self.voting_user1)
+        submit_errors = with_confirmation.forms['student-vote-form'].submit()
```

Comment:
I'm a bit confused that codecov says these lines were not covered when for me, it looks like they should be directly run during a test. Is this a bug in codecov?

Reply:
No, I have used the wrong command and used `coverage | codecov` instead of `coverage && codecov`; now it looks to correctly detect the “coverage”